gitbook-docs/getting-started/installation.md

```diff
@@ -12,6 +12,14 @@ apt update
 apt install -y zkllvm cmake libboost-all-dev
 ```
 
+For Ubuntu, use the following commands:
```

Comment:
It's not about Ubuntu, but rather about running under an unprivileged user. In most cases it means that we're not in a container.

It's supposed that devs either run these commands under a privileged user or use `sudo` when it's needed.

But if you feel it's better to add `sudo` explicitly, please amend the original instruction instead.

Reply:
Yes, this is not Ubuntu specific, rather it covers most Debian use cases. The goal was just to make a sure a developer can copy-paste commands to install - right now this is not the case.

It's highly unusual for any *nix distro for a user to run a terminal under a priveledged user by default. At the same time, using `su` before executing a series of commands is less secure because sudo offers more granular control over user permissions. But a developer cannot just prefix the commands in the doc with `sudo`, e.g. this doesn't work


``` bash
sudo echo 'deb [trusted=yes]  http://deb.nil.foundation/ubuntu/ all main' >>/etc/apt/sources.list
```

and has to be changed to:

```bash
sudo bash -c "echo 'deb [trusted=yes]  http://deb.nil.foundation/ubuntu/ all main' >>/etc/apt/sources.list"
```

I'll change the orginal instructions